Candidate: CVE-2008-4099
PublicDate: 2008-09-18 17:59:00 UTC
References:
 https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-4099
Description:
 CVE-2008-4099 python-PyDNS: insecure DNS transaction ids
Ubuntu-Description:
Notes:
 kees> This issue may have already been solved with other uploads.
Bugs:
Priority: low
Discovered-by:
Assigned-to:
CVSS: 

Patches_python-dns:
upstream_python-dns: released (2.3.1-5)
dapper_python-dns: ignored (reached end-of-life)
feisty_python-dns: needs-triage (reached end-of-life)
gutsy_python-dns: needs-triage (reached end-of-life)
hardy_python-dns: ignored (reached end-of-life)
intrepid_python-dns: not-affected (2.3.3-2)
jaunty_python-dns: not-affected (2.3.3-2)
karmic_python-dns: not-affected (2.3.3-2)
lucid_python-dns: not-affected (2.3.3-2)
maverick_python-dns: not-affected (2.3.3-2)
natty_python-dns: not-affected (2.3.3-2)
oneiric_python-dns: not-affected (2.3.3-2)
devel_python-dns: not-affected (2.3.3-2)
